### PR TITLE
Remove MODNET model

### PR DESCRIPTION
**Pull request by royshil**
The MODNET model is heavy in size (20Mb) and it's accuracy is low compared to the others.

we have pretty good models already, so we can cut back

closing #206

### CODE REVIEW

File: cmake/BuildMyOnnxruntime.cmake
Comment (umireon) [inline cmake/BuildMyOnnxruntime.cmake:1]: I suppose your changes in BuildMyOnnxruntime.cmake would break the Linux build again.

**Review by umireon — CHANGES_REQUESTED** [review]

File: cmake/BuildMyOnnxruntime.cmake
Comment (umireon) [reply cmake/BuildMyOnnxruntime.cmake:1]: libonnxruntime_session.a must come first in CMake.

Comment (umireon) [reply cmake/BuildMyOnnxruntime.cmake:1]: I've confirmed that these changes in BuildMyOnnxRuntime.cmake broke the Linux build.

スクリーンショット 2023-03-24 16 53 45

**Comment by royshil** [comment]
@umireon i was mistaken. i accidentally checked in the Cmake changes - where i shouldnt
but i've reverted hthem now
can you check it out please?

**Review by umireon — APPROVED** [review]
LGTM!

**Pull request merged**